### PR TITLE
Homework #1

### PR DESCRIPTION
**Pull request by minyoungdo**
The assignment this week hasn't been too difficult, but learning how to update the readme file was new to me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+  
+![Me and My Son](PIC.jpg)
+
+![123](bath.jpg)
+
+__... Some more pictures of us__
+
+![Me and My Son](snow.jpg)
+
+![12](cutie.jpg)
```
Comment (YixiC94) [inline README.md:30]: Nice photos!

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+__Another fun fact__ People tell me they can't tell if I am drunk or not because I am always hyper. #too_much_information
+
+![1](drunk_me.jpg)
+
+
+Am I petting my dog, or is *he* petting me?!
+
+![](https://media.giphy.com/media/mOxCUSoRZ7vDq/giphy.gif)
```
Comment (YixiC94) [inline README.md:39]: Great link here!

File: README.md [hunk 3]
```diff
@@ -1 +1,41 @@
-# hw01
```
Comment (YixiC94) [inline README.md:1]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        X   |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |       X    |              |            |
| **Ease of access**          |       X    |              |            |

**Review by YixiC94 — COMMENTED** [review]
Great insertion of links and photos here! Clear structure and easy to follow.

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* You definitely went beyond the expectation for this HW. Great job :)
* I am glad that you have fun with the assignment!

File: README.md (hunk 2 above)
Comment (Jade-Zhou) [inline README.md:39]: That's a good one!

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* You definitely went beyond the expectation for this HW. Great job :)
* I am glad that you have fun with the assignment!

File: README.md [hunk 4]
```diff
@@ -1 +1,41 @@
...
\ No newline at end of file
+# hw01
+
+Hey, this is Minyoung, and I have the cutest dog in the world.
+
+### *My name is __Minyoung Do__*
```
Comment (wanitchayap) [inline README.md:5]: Great combination between italic and header!

File: README.md [hunk 5]
```diff
@@ -1 +1,41 @@
...
\ No newline at end of file
+# hw01
+
+Hey, this is Minyoung, and I have the cutest dog in the world.
+
+### *My name is __Minyoung Do__* 
+#### Nice to meet you!
+
+
+I am from Seoul, South Korea. Before this program, I worked at a local non-profit for a year after graduating from the University of Minnesota. I am so excited to learn more computational skills, meet new people, and try all the good restaurants in Chi-town!
```
Comment (wanitchayap) [inline README.md:9]: Check "Aroy Thai" at North side of Chicago

File: README.md [hunk 6]
```diff
@@ -1 +1,41 @@
...
+![Me and My Son](snow.jpg)
+
+![12](cutie.jpg)
+
+__Another fun fact__ People tell me they can't tell if I am drunk or not because I am always hyper. #too_much_information
+
+![1](drunk_me.jpg)
+
+
+Am I petting my dog, or is *he* petting me?!
```
Comment (wanitchayap) [inline README.md:37]: Really *love* how you underscore words throughout your bio

**Pull request closed without merging**